### PR TITLE
feat(TextLink): accesibility improvements

### PR DESCRIPTION
**Pull request by atabel**
WEB-1623

### CODE REVIEW

File: src/theme.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const TEXTS_ES = {
     disablePasswordVisibility: 'Ocultar contraseña',
     loading: 'Cargando',
     linkOpensInNewTab: 'Se abre en ventana nueva',
+    linkOpensInCurrentPage: 'página actual',
```
Comment (atabel) [inline src/theme.tsx:16]: waiting for real translations:
https://jira.tid.es/browse/LOC-1503

File: src/touchable.tsx [hunk 2]
```diff
@@ -136,6 +136,7 @@ const RawTouchable = React.forwardRef<TouchableElement, Props>((props, ref) => {
     const type = props.type ? props.type : 'button';
 
     const openNewTab = !!props.href && !!props.newTab;
+    const openInCurrentPage = props.href?.startsWith('#');
```
Comment (atabel) [inline src/touchable.tsx:139]: I think this check is enough, wdyt?

File: src/__tests__/text-link-test.tsx [hunk 3]
```diff
@@ -44,3 +44,44 @@ test('TextLink can track events', async () => {
     });
     expect(logEventSpy).toHaveBeenCalledTimes(2);
 });
+
+test('TextkLink indicates that opens in a new tab', async () => {
```
Comment (pladaria) [inline src/__tests__/text-link-test.tsx:48]: typo: `TextkLink`

File: src/__tests__/text-link-test.tsx [hunk 4]
```diff
@@ -44,3 +44,44 @@ test('TextLink can track events', async () => {
...
+        </ThemeContextProvider>
+    );
+
+    const link = await screen.findByRole('link', {name: 'my link Se abre en ventana nueva'});
+
+    expect(link).toBeInTheDocument();
+    expect(link).toHaveAttribute('target');
+});
+
+test('TextkLink indicates that opens in the same page', async () => {
```
Comment (pladaria) [inline src/__tests__/text-link-test.tsx:63]: same

**Review by pladaria — APPROVED** [review]

**Review by marcoskolodny — APPROVED** [review]

**Review by aweell — APPROVED** [review]

**Pull request merged**